collector/tcpstat.go

```diff
@@ -0,0 +1,210 @@
+// +build !notcpstat
+
+package collector
+
+import (
+	"bufio"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+const (
+	procTcpStat  = "/proc/net/tcp"
```

Comment:
Wasn't mature in what sense? It was undergoing breaking changes (unlikely with kernel interfaces), or it didn't work correctly yet? Or just not available in older kernels?